base/process.jl

```diff
@@ -140,7 +145,44 @@ end
 ignorestatus(cmd::Cmd) = Cmd(cmd, ignorestatus=true)
 ignorestatus(cmd::Union{OrCmds,AndCmds}) =
     typeof(cmd)(ignorestatus(cmd.a), ignorestatus(cmd.b))
-detach(cmd::Cmd) = Cmd(cmd, detach=true)
+
+"""
+    detach(command)
+
+Mark a command object so that it will be run in a new process group, allowing it to outlive the julia process, and not have Ctrl-C interrupts passed to it.
+"""
+detach(cmd::Cmd) = Cmd(cmd; flags = cmd.flags | UV_PROCESS_DETACHED)
+
+"""
+    windows_verbatim(command)
+
+In Windows only, mark a command object so that its command-line arguments will *not*
+be quoted or escaped in any way; on other systems, this function does nothing.
+
+On Windows, program arguments are passed as one long "command-line" string,
+and programs are responsible for parsing this string into individual arguments.
+By default, empty arguments or arguments containing spaces or tabs are
+surrounded by double quotes `"` when they are passed, and `"`' and `\` characters
+in the arguments are preceded by a `\` (escaped).   `windows_verbatim`
+causes the arguments to be pasted verbatim into the command-line string instead,
+and is used for programs that do not handle quotes and escapes in their
+command-line parsing.
+"""
+windows_verbatim(cmd::Cmd) = Cmd(cmd; flags = cmd.flags | UV_PROCESS_WINDOWS_VERBATIM_ARGUMENTS)
+
+"""
+    windows_hide(cmd)
+
+On Windows only, hide the subprocess console window that might otherwise
+be created when `cmd` is run; on other systems this function does nothing.
+
+On Windows, if Julia itself is running in a console window, then the
+output of `cmd` appears in the same console.  However, if Julia is running
+without a console window, e.g. as a background process, then running
+`cmd` will normally create a new console window to display its output.
+The `windows_hide` function suppresses that console window.
+"""
+windows_hide(cmd::Cmd) = Cmd(cmd; flags = cmd.flags | UV_PROCESS_WINDOWS_HIDE)
```

Comment:
rather than exposing this as a new function, I was considering making this an official API for Cmd:
```julia
Cmd(cmd; detach = false, windows_hide = true)
```

thoughts?